### PR TITLE
Remove port serdes object before removing port

### PR DESCRIPTION
**Pull request by kcudnik**
**What I did**
Added removal of port serdes objects if exists, before removing port.

**Why I did it**
If port serdes is present on device, it's a reference dependency on port.

**How I verified it**
locally

**Details if related**

### CODE REVIEW

**Comment by kcudnik** [comment]
This PR is needed before https://github.com/Azure/sonic-sairedis/pull/985, since remove port without remove port serdes first will cause failure in port remove with  SAI_STATUS_OBJECT_IN_USE error code and will fail unittest 
```
test_port_config.py::TestPortConfig::test_port_breakout FAILED           [ 66%]
```
test details here: https://dev.azure.com/mssonic/build/_build/results?buildId=74040&view=logs&j=3f6395b2-1619-5ebe-f305-2aedcf353cb5&t=f9db2383-9349-50ba-61c3-4bde15cec59e

**Comment by kcudnik** [comment]
LGTM failure is not related to PR, its related to doxygen break backward config compatibility in SAI repo

File: orchagent/portsorch.cpp [hunk 1]
```diff
@@ -2289,6 +2289,13 @@ sai_status_t PortsOrch::removePort(sai_object_id_t port_id)
     }
     /* else : port is in default state or not yet created */
 
+    /*
+     * Remove port serdes (if exists) before removing port since this is
```
Comment (prsunny) [inline orchagent/portsorch.cpp:2293]: typo - is

**Review by prsunny — DISMISSED** [review]

**Comment by kcudnik** [comment]
@prsunny  all tests passed, but because of LGTM errors, i can't merge this, this needs to be force merged, and i don't have enough privilege's on this repo

**Review by prsunny — APPROVED** [review]

**Pull request merged**